### ISSUE TITLE
Sensible size of input forms in settings

### ISSUE THREAD

**Issue by ingoj** [friendica/friendica#8763]
Hi!

In Settings some settings are a little bit awkward to use, because the input form is a bit small:

<img width="846" alt="Bildschirmfoto 2020-06-12 um 11 07 03" src="https://user-images.githubusercontent.com/18173881/84485906-ed0ab900-ac9c-11ea-8d8e-0c822a2475ac.png">

I think it would be good to have these input forms at a reasonable size, i.e. the same width as the floating text underneath. 

The worst setting is Register Text, which ist just a few characters wide: 
<img width="780" alt="Bildschirmfoto 2020-06-12 um 11 08 57" src="https://user-images.githubusercontent.com/18173881/84486117-3529db80-ac9d-11ea-8516-745a9148e96a.png">

Similar is the situation with Allowed friend domains and allowed mail domains: here I would rather expect a scrollable list form and an input field to add new domains. 

Of course this might be different in other themes than "Vier", though...

**Comment by MrPetovan** [comment]
Yes, it is a theme-specific issue, if anyone wants to tackle it and verify most pages with input fields aren't negatively affected by an increase in size.

**Comment by AlfredSK** [comment]
There are also some very tiny input fields in add-on settings like 'nsfw'.
